default.cfg.example

```diff
@@ -132,7 +132,8 @@ hideCoins = True
 #ReportInterval = 86400
 
 #[CHARTS]
-#DumpInterval=21600
+#DumpInterval = 21600
+#HistoryFile = www/history.json
```

Comment:
Naming convention, lower cased + underscore

Reply:
What specifically? I was going off AccountStats variable ReportInterval convention.

Reply:
Well, you also introduced 'ReportInterval'  and actually they all should be lowercase. :)